secure-value-exchange/quickstart.md

```diff
@@ -0,0 +1,252 @@
+# Quickstart
+
+This guide aims to get you up and running and familiar with the Exchange API in about 15 minutes. We'll teach you how to
+
+- create and manage DID
+- issue and revoke Verifiable Credentials
+
+**_NOTE:_** It assumes you already have an approved account in dev.meeco.me portal and subscription key to access exchange API
+
+---
+
+## Create a DID for holder
+
+```curl
+
+curl -v -X POST "https://sandbox.meeco.me/exchange/dids/hedera"-H "Cache-Control: no-cache"-H "Meeco-Subscription-Key: replace-it-with-your-subscription-key"
```

Comment:
Can you use multiline curl commands so that the headers are not cut off? Also, do you need the cache control header?

Reply:
not sure how to get header and body with curl POST

Reply:
No I mean type it like

Reply:
for some reason multi-line do not work for me.

Reply:
yah added now multiline